src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -122,111 +122,119 @@
+All parameters are case-insensitive.
 
--Interactive
-    Present an interactive prompt to the user.
+-Command | -c
+    Executes the specified commands (and any parameters) as though they were typed
+    at the PowerShell command prompt, and then exits, unless NoExit is specified.
+    The value of Command can be "-", a string. or a script block.
 
--InputFormat
-    Describes the format of data sent to PowerShell. Valid values are
-    "Text" (text strings) or "XML" (serialized CLIXML format).
+    If the value of Command is "-", the command text is read from standard input.
 
--OutputFormat
-    Determines how output from PowerShell is formatted. Valid values
-    are "Text" (text strings) or "XML" (serialized CLIXML format).
+    If the value of Command is a script block, the script block must be enclosed
+    in braces ({}). You can specify a script block only when running 'powershell'
+    in PowerShell. The results of the script block are returned to the
+    parent shell as deserialized XML objects, not live objects.
 
--WindowStyle
-    Sets the window style to Normal, Minimized, Maximized or Hidden.
+    If the value of Command is a string, Command must be the last parameter in the command,
+    because any characters typed after the command are interpreted as the command arguments.
 
--EncodedCommand
-    Accepts a base-64-encoded string version of a command. Use this parameter
-    to submit commands to PowerShell that require complex quotation
-    marks or curly braces.
+    To write a string that runs a PowerShell command, use the format:
+    "&amp; {&lt;command&gt;}"
+    where the quotation marks indicate a string and the invoke operator (&amp;)
+    causes the command to be executed.
+
+    Example:
+        powershell -Command {Get-WinEvent -LogName security}
+        powershell -command "&amp; {Get-WinEvent -LogName security}"
 
--ConfigurationName
+-ConfigurationName | -config
     Specifies a configuration endpoint in which PowerShell is run.
-    This can be any endpoint registered on the local machine including the
-    default PowerShell remoting endpoints or a custom endpoint having
-    specific user role capabilities.
-
--File
-    Runs the specified script in the local scope ("dot-sourced"), so that the
-    functions and variables that the script creates are available in the
-    current session. Enter the script file path and any parameters.
-    File must be the last parameter in the command, because all characters
-    typed after the File parameter name are interpreted
-    as the script file path followed by the script parameters.
+    This can be any endpoint registered on the local machine including the default PowerShell
+    remoting endpoints or a custom endpoint having specific user role capabilities.
+
+    Example: powershell -configurationmame AdminRoles
 
--ExecutionPolicy
+-EncodedCommand | -e | -ec
+    Accepts a base-64-encoded string version of a command. Use this parameter to submit
```

Comment:
I think we should use: base64 encoded

Without the hyphens

Reply:
Fixed.